src/client/components/Molecules/Navbar/Navbar.jsx

```diff
@@ -0,0 +1,31 @@
+import css from './Navbar.module.css';
+import { NavigationLink } from 'client/components/Atoms/NavigationLink/NavigationLink';
+
+const Navbar = () => {
+  return (
+    <nav className={css.navigation}>
+      <ul className={css.list}>
+        <li className={css.item}>
+          <NavigationLink path="/categories/:categoryName" title="Categories" />
+        </li>
+        <li className={css.item}>
+          <NavigationLink path="/add" title="Add recipes" />
+        </li>
+        <li className={css.item}>
+          <NavigationLink path="/my" title="My recipes" />
+        </li>
+        <li className={css.item}>
+          <NavigationLink path="/favorite" title="Favorite" />
+        </li>
+        <li className={css.item}>
+          <NavigationLink path="/shopping-list" title="Shopping list" />
+        </li>
+        <li className={css.item}>
+          <NavigationLink path="/search" title="Search" icon="true" />
+        </li>
+      </ul>
```

Comment:
I had the same idea but in my opinion the problems are as follow:
1) paths are not the same as titles, for example path="/my" title="My recipes"
2) one of list items has and additional props "icon".
If we had more time, I would have tried to do that, but I suggest maintaining it in this shape now - do you agree with me?

Reply:
Then in your template use switch case and just pass only category name from your list and rest will be described in your switch case template.

Reply:
To be honest, I was trying to do it following your tips, but it hadn't worked. Instead of this, I applied an array of objects with map function to render nav links. Check it, please, weather it is correct.

Reply:
✅